src/ReporterOrchestrator.ts

```diff
@@ -0,0 +1,48 @@
+import {StrykerOptions} from './api/core';
+import {Reporter, ReporterFactory} from './api/report';
+import ClearTextReporter from './reporters/ClearTextReporter';
+import ProgressReporter from './reporters/ProgressReporter';
+import BroadcastReporter, {NamedReporter} from './reporters/BroadcastReporter';
+import * as log4js from 'log4js';
+
+const log = log4js.getLogger('ReporterOrchestrator');
+
+
+export default class ReporterOrchestrator {
+
+  constructor(private options: StrykerOptions) {
+    this.registerDefaultReporters();
+  }
+
+  public createSingleReporter(): Reporter {
```

Comment:
Yeah.. but than i would expect it to return a `BroadcastReporter` and that one actually doesn't implement the methods statically as far as typescript is concerned. But it would be the best name, agreed. I will change it.